core/os/input.h

```diff
@@ -100,10 +100,29 @@ class Input : public Object {
 	virtual int    get_joy_button_index_from_string(String p_button)=0;
 	virtual int    get_joy_axis_index_from_string(String p_axis)=0;
 
+	enum TrackerType {
+		TRACKER_ANY, /* used by get_connected_trackers to return all types */
+		TRACKER_UNKNOWN, /* unknown tracker */
+		TRACKER_HMD, /* HMD in the broad sense, mobile device position is typed like this too */
+		TRACKER_CONTROLLER, /* tracks a controller */
+		TRACKER_BASESTATION /* tracks location of a base station */
```

Comment:
Hmmm, indeed I wonder about the use.. In reality you'd probably just be selecting TRACKER_HMD first, and TRACKER_CONTROLLER second, and ignore the rest.